wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -291,16 +291,58 @@ $( document ).ready(function() {
     jQuery(document).on('click', '#sortreverse', function(e) {
         e.preventDefault()
 
-        jQuery('.arcsi-blokks').html(jQuery('.arcsi-blokks').find(jQuery('.arcsi-blokks .arcsiblokk')).get().reverse());
+        const mylist = $('.arcsi-blokks');
+        let listitems = mylist.children(".arcsiblokk");
 
         if (ascending) {
             ascending = false
```

Comment:
You introduce here alphabetascending later down so I think we shoul rename this too to dateascending? In order to clarify what ascending it is for the future